dist/index.html

```diff
@@ -45,42 +45,78 @@ <h1 class = "title">What's Cookin?</h1>
   </main>
   <div class="all-recipes-wrapper">
     <aside class="all-filters hidden">
-      <input type="checkbox" class="checkbox" id="antipasto" name="checkbox">
-      <label for="antipasto">Antipasto</label>
-      <input type="checkbox" class="checkbox" id="appetizer" name="checkbox">
-      <label for="appetizer">Appetizer</label>
-      <input type="checkbox" class="checkbox" id="breakfast" name="checkbox">
-      <label for="breakfast">Breakfast</label>
-      <input type="checkbox" class="checkbox" id="brunch" name="checkbox">
-      <label for="brunch">Brunch</label>
-      <input type="checkbox" class="checkbox" id="condiment" name="checkbox">
-      <label for="condiment">Condiment</label>
-      <input type="checkbox" class="checkbox" id="dinner" name="checkbox">
-      <label for="dinner">Dinner</label>
-      <input type="checkbox" class="checkbox" id="dip" name="checkbox">
-      <label for="dip">Dip</label>
-      <input type="checkbox" class="checkbox" id="hor d'oeuvre" name="checkbox">
-      <label for="hor d'oeuvre">Hor d'oeuvre</label>
-      <input type="checkbox" class="checkbox" id="lunch" name="checkbox">
-      <label for="lunch">Lunch</label>
-      <input type="checkbox" class="checkbox" id="main course" name="checkbox">
-      <label for="main course">Main Course</label>
-      <input type="checkbox" class="checkbox" id="main dish" name="checkbox">
-      <label for="main dish">Main Dish</label>
-      <input type="checkbox" class="checkbox" id="morning meal" name="checkbox">
-      <label for="morning meal">Morning Meal</label>
-      <input type="checkbox" class="checkbox" id="salad" name="checkbox">
-      <label for="salad">Salad</label>
-      <input type="checkbox" class="checkbox" id="sauce" name="checkbox">
-      <label for="sauce">Sauce</label>
-      <input type="checkbox" class="checkbox" id="side dish" name="checkbox">
-      <label for="side dish">Side Dish</label>
-      <input type="checkbox" class="checkbox" id="snack" name="checkbox">
-      <label for="snack">Snack</label>
-      <input type="checkbox" class="checkbox" id="spread" name="checkbox">
-      <label for="spread">Spread</label>
-      <input type="checkbox" class="checkbox" id="starter" name="checkbox">
-      <label for="starter">Starter</label>
+      <div class="filter-option">
+        <input type="checkbox" class="checkbox" id="antipasto" name="checkbox">
+        <label for="antipasto">Antipasto</label>
+      </div>
```

Comment:
Nice job wrapping the checkbox input and label in a div.

Reply:
This looks really good!